.github/workflows/pr-push.yml

```
@@ -0,0 +1,24 @@
+# Workflow based on the w3c/spec-prod action example to deploy to W3C using Echidna:
+# https://github.com/w3c/spec-prod/blob/main/docs/examples.md#deploy-to-w3c-using-echidna
+
+name: CI
+
+on:
+  pull_request: {}
+  push:
+    branches: [gh-pages]
+
+jobs:
+  main:
+    name: Build, Validate and Deploy
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v2
+      - uses: w3c/spec-prod@v2
+        with:
+          W3C_ECHIDNA_TOKEN: ${{ secrets.ECHIDNA_TOKEN }}
+          # Replace following with appropriate value. See options.md for details.
+          W3C_WG_DECISION_URL: https://www.w3.org/2021/04/07-dap-minutes.html#r01
+          W3C_BUILD_OVERRIDE: |
```

Just added the following too in the settings ... will send me an email when we publish: 

```suggestion
          W3C_NOTIFICATIONS_CC: ${{ secrets.CC }}
          W3C_BUILD_OVERRIDE: |
```